lib/FusionInventory/Agent/Task/Inventory/Win32/Softwares.pm

```diff
@@ -411,4 +430,56 @@ sub _processMSIE {
 
 }
 
+# List of SQL Instances
+sub _getSqlInstancesList {
+    my (%params) = @_;
+    my $logger = $params{logger};
+	my $softwareName = $params{softwarename};
+	my $softwareVersion = $params{softwareversion};
+	
+	# Registry access for SQL Instances
+    my $machKey = $Registry->Open('LMachine', {Access => KEY_READ}) or $logger->error("Can't open HKEY_LOCAL_MACHINE key: $EXTENDED_OS_ERROR");
```

Comment:
Please, rebase your branch to 2.4.x branch and don't use anymore $Registry->Open() syntax. We have a getRegistryKey() API from FusionInventory::Agent::Tools::Win32 and you must now use it.

Again, don't mix spaces & tabulation indentation, just use spaces.